orchard/orchard_client.go

```diff
@@ -86,6 +86,33 @@ func (c OrchardRestClient) create(result string) (string, error) {
 	return string(orchardID), nil
 }
 
+func (c OrchardRestClient) IsActivated(orchardID string) (bool, error) {
```

Comment:
Let's make it to `Details` and return parsed details instead. Whoever is calling this should deal with what status it returns